src/Core.UnitTests/SystemAbstractions/ProcessWrapperTests.cs

```diff
@@ -18,24 +18,235 @@
  * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
  */
 
+
+using System;
 using System.Diagnostics;
+using System.IO;
+using System.Text;
 using FluentAssertions;
+using Microsoft.Internal.VisualStudio.PlatformUI;
 using Microsoft.VisualStudio.TestTools.UnitTesting;
 using SonarLint.VisualStudio.Core.SystemAbstractions;
+using SonarLint.VisualStudio.Integration.UnitTests.Helpers;
 
 namespace SonarLint.VisualStudio.Core.UnitTests.SystemAbstractions
 {
     [TestClass]
     public class ProcessWrapperTests
     {
+        public TestContext TestContext { get; set; }
+
         [TestMethod]
-        public void GetCurrentProcessMainWindowHandle_ReturnsCurrent()
+        public void ProcessFactory_Start_ReturnsWrappedProcess()
```

Comment:
GitHub is getting confused here:
* the old `Core.ProcessWrapperTests` was renamed to `Core.CurrentProcessWrapperTests`
* this class was moved from `CFamily` and the namespace changed.

The code and tests haven't changed.